.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "pybind11/pybind11"]
```

Comment:
In the current format yes. But I want to put the package description directly into the original repo. It will have to be a submodule then.